var/spack/repos/builtin/packages/net-snmp/package.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class NetSnmp(AutotoolsPackage):
+    """SNMP Daemon."""
+
+    homepage = "http://net-snmp.sourceforge.net"
+    url      = "https://udomain.dl.sourceforge.net/project/net-snmp/net-snmp/5.9/net-snmp-5.9.tar.gz"
```

Comment:
It is preferable to *not* go through a web hosting company.

The source -- and version you explicitly provide -- appears to be available at  https://github.com/net-snmp/net-snmp.  

Alternatively, the `url` should perhaps be off `https://sourceforge.net/projects/net-snmp/files/` or `https://sourceforge.net/p/net-snmp/code/ci/master/tree/` (the repository shows https and git clone options).